src/js/stores/AppsStore.js

```diff
@@ -295,7 +314,7 @@ QueueStore.on(QueueEvents.CHANGE, function () {
 AppDispatcher.register(function (action) {
   switch (action.actionType) {
     case AppsEvents.REQUEST_APPS:
-      storeData.apps = processAppsAndEmptyGroups(action.data.body);
+      storeData.apps = processAppsAndGroups(action.data.body);
```

Comment:
@orlandohohmeier nope – that's what I am complaining about. `processAppsAndGroups` actually returns only `apps` when it's done recursing.